modules/profile/README.md

```diff
@@ -0,0 +1,90 @@
+# SCC Profile Module
+
+This module creates an SCC Profile (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-build-custom-profiles&interface=ui). A profile is a grouping of controls that can be evaluated for compliance.
+
+The module supports the following actions:
+- Create SCC Profile
+
+### Usage
+
+```hcl
+provider "ibm" {
+  ibmcloud_api_key = "XXXXXXXXXX" # pragma: allowlist secret
+  region           = "us-south"
+}
+
+# - SCC Profile
+module "create_scc_profile" {
+  source           = "terraform-ibm-modules/scc/ibm//modules/profile"
+  instance_id = module.create_scc_instance.guid
```

Comment:
use a dummy value like whats in provider doc `"00000000-1111-2222-3333-444444444444"`